sdk/purview/azure-purview-workflow/README.md

```diff
@@ -0,0 +1,94 @@
+# Azure Purview Workflow Service client library for Python
+
+Workflows are automated, repeatable business processes that users can create within Microsoft Purview to validate and orchestrate CUD (create, update, delete) operations on their data entities. Enabling these processes allow organizations to track changes, enforce policy compliance, and ensure quality data across their data landscape.
```

Comment:
curious: why is read left out of `CRUD`?

Reply:
Read is not an operation which will change data entities